aten/src/ATen/native/Sorting.cpp

```diff
@@ -412,97 +452,135 @@ Tensor median_impl(const Tensor& self, bool ignore_nan) {
 } // namespace
 
 Tensor& quantile_out(
-    Tensor& out,
     const Tensor& self,
     const Tensor& q,
     optional<int64_t> _dim,
-    bool keepdim) {
-  quantile_impl(out, self, q, std::move(_dim), keepdim, /*ignore_nan=*/false);
+    bool keepdim,
+    const std::string interpolation,
+    Tensor& out) {
+  quantile_impl(
+      out,
+      self,
+      q,
+      _dim,
+      keepdim,
+      get_quantile_interpolation_mode(interpolation),
+      /*ignore_nan=*/false);
   return out;
 }
 
 Tensor& quantile_out(
-    Tensor& out,
     const Tensor& self,
     double q,
     optional<int64_t> _dim,
-    bool keepdim) {
+    bool keepdim,
+    const std::string interpolation,
+    Tensor& out) {
   TORCH_CHECK(
       q >= 0 && q <= 1, "quantile() q must be in the range [0, 1] but got ", q);
   return at::quantile_out(
       out,
       self,
       at::scalar_tensor(q, self.options()),
-      std::move(_dim),
-      keepdim);
+      _dim,
```

Comment:
This lost a std::move, which isn't a big deal -- any reason why?

Reply:
optional<int64_t> is a trivially copyable value so std::move has no effect.